src/main/java/com/exactpro/th2/crawler/Crawler.java

```diff
@@ -198,25 +200,34 @@ public Duration process() throws IOException, UnexpectedDataProcessorException {
                 Timestamp startTime = toTimestamp(interval.getStartTime());
                 Timestamp endTime = toTimestamp(interval.getEndTime());
                 do {
-                    LOGGER.trace("Requesting data for interval");
-                    data = requestData(startTime, endTime, continuation, parameters);
-                    continuation = data.getContinuation();
-                    var currentData = data;
-                    long remaining = sendingReport == null ? 0 : sendingReport.getRemainingData();
-                    sendingReport = currentData.getHasData()
-                            ? processData(currentInt, parameters, currentData)
-                            : requireNonNullElse(sendingReport, Report.empty());
-
-                    if (currentData.getHasData()) {
-                        metrics.updateProcessedData(crawlerType, currentData.size());
-                    }
+                    if (parameters.getSessionAliases().isEmpty()) {
+                        LOGGER.debug("Interval from {}, to {} does not contain messages with appropriate aliases",
+                                interval.getStartTime(), interval.getEndTime());
+                        data = new CrawlerDataStub();
+                        sendingReport = Report.empty();
```

Comment:
Do we need all this? Can't we simply set sending report to `Report.empty();` and break the loop?